src/win/pipe.c

```diff
@@ -1381,11 +1390,6 @@ static int uv__pipe_write_data(uv_loop_t* loop,
     handle->write_queue_size += req->u.io.queued_bytes;
   } else if (handle->flags & UV_HANDLE_BLOCKING_WRITES) {
     /* Using overlapped IO, but wait for completion before returning */
-    req->u.io.overlapped.hEvent = CreateEvent(NULL, 1, 0, NULL);
```

Comment:
Looks like we were creating an event with the manual reset flag sset before, but I see no call to ResetEvent, so was this a bug then?

Reply:
No, you're looking at the wrong commit. This is just supposed to consolidate the repeated code. I see what you mean though about the parameter change there, we are reseting this by calling `CloseHandle`, but it does bring up some other bugs though, so I'll address those.

Reply:
Gotcha, thanks for explaining!